### PR TITLE
v0.0.19

### PR DESCRIPTION
**Pull request by brijeshb42**
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/pigment-css/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,33 @@
...
 
+## 0.0.19
+
+<!-- generated comparing v0.0.18..master -->
+
+_Aug 7, 2024_
+
+A big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```
Comment (mnajdova) [inline CHANGELOG.md:14]:
```suggestion
A big thanks to the 3 contributors who made this release possible.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,33 @@
...
+### `@pigment-css/react@0.0.19`
+
+- &#8203;<!-- 6 -->[react] Fix thrown error during eval phase (#196) @brijeshb42
+- &#8203;<!-- 1 -->[TypeScript] Fix `sx` types and missing `style` prop from layout components (#191) @siriwatknp
+
+### Docs
+
+- &#8203;<!-- 4 -->[docs] Clarify security policy @oliviertassinari
+- &#8203;<!-- 3 -->[docs] Clarify security policy @oliviertassinari
+- &#8203;<!-- 2 -->[docs] Clarify security policy @oliviertassinari
```
Comment (mnajdova) [inline CHANGELOG.md:25]:
```suggestion
- &#8203;<!-- 4 -->[docs] Clarify security policy @oliviertassinari
```

**Review by mnajdova — COMMENTED** [review]
Left comments on the changelog. Why do we have some pnpm-lock.yaml changes?

**Comment by brijeshb42** [comment]
@mnajdova Here's a PR to fix the issues with this one - https://github.com/mui/pigment-css/pull/198

**Pull request merged**